README.md

```diff
@@ -1,3 +1,273 @@
+
+<pre>
+landscape:
+  <a href="#landscapename">name</a>: &lt;URL-friendly name&gt;                       # Gardener managed k8s landscape name
+
+  <a href="#landscapecluster">cluster</a>:                                          # Information about your base cluster
+    kubeconfig: &lt;relative path + filename&gt;          # Path to your `kubeconfig` file, rel. to folder `landscape`
+    domain: &lt;prefix&gt;.&lt;cluster domain&gt;               # Unique basis domain for DNS entries
+    iaas: &lt;gcp|aws|azure&gt;                           # iaas provider (coming soon: openstack|alicloud)
+    region: &lt;major region&gt;-&lt;minor region&gt;           # Example: europe-west1
+
+  <a href="#landscapenetworks">networks</a>:                                         # <a target="_blank" rel="noopener noreferrer" href="https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing">CIDR IP ranges</a> of base cluster
+    nodes: &lt;CIDR IP range&gt;                                  
+    pods: &lt;CIDR IP range&gt;                                   
+    services: &lt;CIDR IP range&gt;                             
+
+  <a href="#landscapeiaas">iaas</a>:
+    region: &lt;major region&gt;-&lt;minor region&gt;      # region that Gardener will use for seed clusters
+    zones:
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-b
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-c     
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-d
+    credentials:                               # credentials to get access to the seed cluster through service account
+
+  <a href="#landscapeetcd">etcd</a>:
+    backup:
+      type: &lt;gcs|s3&gt;                    # type of blob storage
+      region: (( iaas.region ))         # region of blob storage (default: same as above)
+      credentials: (( iaas.creds ))     # credentials for the blob storage's IaaS provider (default: same as above)
+
+  <a href="#landscapedns">dns</a>:
+    type: &lt;google-clouddns|aws-route53&gt;       # dns provider
+    hostedZoneID:                             # hosted zone id for chosen domain
+    credentials:                              # credentials for the dns provider
+
+  <a href="#landscapeidentity">identity</a>:
+    users:
+      - email:                                # email (used for Gardener dashboard login)
+        username:                             # username (displayed in Gardener dashboard)
+        password:                             # clear-text password (used for Gardener dashboard login)
+      - email:                                # see above
+        username:                             # see above
+        hash:                                 # bcrypted hash of password, see above
+</pre>
+
+
```

Comment:
We are currently missing the formats (they yaml keys, to be more precise) in which the credentials need to be specified. These are:

AWS:
region: ...
accessKeyID: ...
secretAccessKey: ...

GCP:
serviceaccount.json: |
  {
    "type": "...",
    "project_id": "...",
    ...
  }

Azure:
clientID: ...
clientSecret: ...
subscriptionID: ...
tenantID: ...

Reply:
done